perl/README.md

```diff
@@ -0,0 +1,64 @@
+# Bystro Perl package
+
+## Installing Bystro using docker
+
+To build a docker image using the `Dockerfile`, run the following:
+
+```bash
+docker build --tag bystro-cli .
+```
+
+## Installing Bystro dependencies locally
+
+Assuming that you've cloned the repository and are working on it locally, then the dependencies can mostly be installed with cpanm.
+But there are a few one-off dependencies that require a slightly modified approach.
+
+One-off dependencies can be installed as follows:
+
+```bash
+cpanm --quiet https://github.com/bystrogenomics/msgpack-perl.git
+cpanm --quiet --notest MouseX::Getopt
+git clone --depth 1 -branch v0.12 --recurse-submodules https://github.com/salortiz/LMDB_File.git \
```

Comment:
-branch v0.12 should be `--branch v0.12`

Reply:
Let's specify versions in the perl code itself at which point this could be updated. Right now, I dropped the branch switch.